### ISSUE TITLE
LLD Fissure skill

### ISSUE THREAD

**Issue by stinks88** [kolton/d2bot-with-kolbot#890]
i found other sources googling. but cant open pages. the skillfissure, picks up carrion vine pelts... any fix to this yet?

**Comment by justinpees** [comment]
check in NTItemAlias file if this is what you have for fissure:

NTIPAliasStat["skillfissure"] = [107,231];

maybe you accidentally changed one of the numbers there. if thats good, then you are probably right that it is a bug that needs to be fixed

**Comment by stinks88** [comment]
Yes. Its matchs what you wrote down. When googled it showed some one else talking about. Carrion vine pelts where getting picked in the pickit.nip and the line showed skillfissure. And im having the same issue. Any way we can fix this bug?

**Comment by justinpees** [comment]
well if they just got them mixed up then you would just change the 231 to "234". try that and then in pickit script use this to pickup any fissure pelt to verify if it worked.

[type] == pelt && [quality] >= normal # [skillfissure] >= 1

**Comment by stinks88** [comment]
Thanks. Im also testing skillcarrionvine underneath it. To see what it picks up under that skill. And ill set it to >= 1

**Comment by justinpees** [comment]
> Thanks. Im also testing skillcarrionvine underneath it. To see what it picks up under that skill. And ill set it to >= 1

mind if i check out your runs? my email is on my profile here if you want me to join in game for some chat

**Comment by justinpees** [comment]
let me know if it is just a mixup in skills

**Comment by stinks88** [comment]
![backwards](https://user-images.githubusercontent.com/46278161/50570814-df42cb80-0d67-11e9-8898-8c718e7e032b.jpg)

Finally found one!. And guess what, its Backwards!. soo found the solution!, Thanks for all the help Justinpees.
